cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -51,13 +52,13 @@
                     block__padded-top
                     block__border-top">
         <h2>Job Description</h2>
-        {{ page.job_description }}
+        {{ page.description | safe }}
     </section>
 
     <section class="block">
         <em>
-            {% if page.job_equal_opportunity_text %}
-                {{ page.job_equal_opportunity_text }}
+            {% if page.equal_opportunity_text %}
+                {{ page.equal_opportunity_text }}
```

Comment:
I don't think so -- the equal opportunity text should show up on all job postings by default.